core/app/core/src/lib/fields/base/base-field.component.ts

```diff
@@ -73,7 +79,12 @@ export class BaseFieldComponent implements FieldComponentInterface, OnInit, OnDe
         const fieldKeys = (this.record.fields && Object.keys(this.record.fields)) || [];
         if (fieldKeys.length > 1) {
             this.calculateDependentFields(fieldKeys);
-            this.field.previousValue = this.field.value;
+            this.previousValue = deepClone({
+                value: this.field.value,
+                valueList: this.field.valueList,
+                valueObject: this.field.valueObject,
+                forceNotEqual: 'forceNotEqual',
```

Comment:
Hi @mpuyosa91, where do we use the `forceNotEqual`?

Reply:
Hi @clemente-raposo is just a way to make it different and force a true in line 120. It also could be accomplished with an undefined due to line 47 but I was anticipating any use of that first state of `this.previousValue`